src/cord/stream.ts

```diff
@@ -75,7 +76,9 @@ export class Stream {
           qstream
         );
 
+       const vc = VCUtils.fromCredential(credential, schema);
```

Comment:
This is working smoothly with example code able to get details of VC in response.